src/main/webapp/js/projects/react/components/ExpandedProject.js

```diff
@@ -0,0 +1,28 @@
+import {expandedProjectType} from '../prop_types.js';
+
+/**
+ * An expanded view of a project.
+ * @param {Object} props
+ * @return {React.Component} Returns the React component.
+ */
+export default function ExpandedProject(props) {
```

Comment:
is it neccesary to use default exports?

Reply:
It's not necessary. It's just React convention, although I'm realizing from Bill's message below that `export default` is not good style according to the [style guide](https://google.github.io/styleguide/jsguide.html#es-module-exports) so I'll be removing this.